_config/security.yml

```diff
@@ -1,3 +1,27 @@
+---
+Name: cwppasswordstrength
+---
+# In the case someone uses `new PasswordValidator` instead of Injector
+# test names will not be set, as it is not configurable.
```

Comment:
FYI we have generally use a 120 char line length

Reply:
Yeah, sorry back punctuation. They're different lines because I'm treating them as 2 points, although they are (heavily) related.